### PR TITLE
Update documentation and testing

### PR DESCRIPTION
**Pull request by L1uY1jun**

### CODE REVIEW

File: docs/diagrams/AddSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@startuml
+!include style.puml
+
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":LinkedoutParser" as LinkedoutParser LOGIC_COLOR
```
Comment (benjytan45678) [inline docs/diagrams/AddSequenceDiagram.puml:7]: Wrong name for Parser

File: docs/diagrams/AddCommandActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@startuml
+start
+:User enters Add command;
+
+if () then ([else])
```
Comment (benjytan45678) [inline docs/diagrams/AddCommandActivityDiagram.puml]: Do u think it is better to first have valid input, applicant already exists in TrackAScholar and then have else for invalid input?

File: docs/diagrams/AddSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+LinkedoutParser -> AddCommandParser : AddCommandParser()
+activate AddCommandParser
+
+AddCommandParser --> LinkedoutParser
+deactivate AddCommandParser
+
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+AddCommandParser -[hidden]-> LinkedoutParser: BUFFER_MESSAGE
+
+LinkedoutParser -> AddCommandParser : parse(add n/Sam \n p/98886767 \n e/sam@example.com \n s/NUS Merit Scholarship \n as/pending)
```
Comment (benjytan45678) [inline docs/diagrams/AddSequenceDiagram.puml:35]: when u parse, does its argument also contain the command word add?

File: docs/diagrams/AddCommandActivityDiagram.puml (hunk 2 above)
Comment (L1uY1jun) [reply docs/diagrams/AddCommandActivityDiagram.puml]: This one is purely for aesthetic purposes as the valid condition extends too long if placed as the first if condition on an activity diagram

**Pull request merged**